src/assets/css/styles.css

```diff
@@ -0,0 +1,247 @@
+.row {
+    display: flex;
+    flex-wrap: nowrap;
+    margin: 0 -15px 15px;
+}
+
+.col {
+    padding: 0 15px;
+}
+
+.p_0_15 {
```

Comment:
I'm not a fan of these kinds of classNames. You'd better use something "more semantic"

Reply:
Agree with Anton. Better use semantic mark-up